packages/react-component-library/src/components/Tooltip/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+
+export interface TooltipProps extends PositionType {
+  id?: string
+  position?: 'above' | 'below' | 'left' | 'right'
+  title?: string
+  width?: number
+}
+
+export const Tooltip: React.FC<TooltipProps> = ({
+  bottom,
+  children,
+  id = '',
+  left,
+  position,
+  right,
+  title,
+  top,
+  width,
+}) => {
+  const style = {
+    bottom,
+    left,
+    right,
+    top,
+    width,
+  }
+
+  return (
+    <div
+      className={`rn-tooltip
+      ${position ? `rn-tooltip--${position}` : ''}
```

Comment:
Is it cleaner to stick to using the classNames utility for these conditional classes?

Reply:
I think it would be even better to set a default for position and use classNames to build this to be consistent.